templates/site/js/login-styles-scripts.js

```diff
@@ -0,0 +1,41 @@
+/*Initialize Popper.js and Tooltips.js*/
+$(document).ready(function() {
+  $('[data-toggle="tooltip"]').tooltip();
+  $('[data-toggle="popover"]').popover();
+
+})
+
+/*Toggle eye icon and password text in password input field*/
+function togglePassword() {
+    var passwordInput = document.getElementById("userPassword");
+
+    if (passwordInput.type === "password") {
+        passwordInput.type = "text";
+    } else {
+        passwordInput.type = "password";
+    }
+}
+
+function toggleEyeIcon(element) {
+    var eyeIcon = element.firstElementChild;
+
+    eyeIcon.classList.toggle("icon-eye");
+    eyeIcon.classList.toggle("icon-eye-slash");
+}
+
+
+window.onload = () => {
+    var userPassword = document.getElementById("userPassword");
+    var eyeIconContainer = userPassword.nextElementSibling;
```

Comment:
It's because the div for the eye icon is in the username textfield as well, it's just hidden. So getting the eye icon div by id gets the hidden one and not the one I want.

I realize now I could have put a condition on the eye icon to include it dependent on the textfield type.

Reply:
The issue is that you have two elements with the id `eyeIconContainer` on the same page (ids *must* be unique)

Reply:
Put the eye icon div inside a condition so it only displays in password field.